pages/techblog/components/techSubComponents.tsx

```diff
@@ -54,21 +61,27 @@ export const TechContent = ({
   maxLines: number;
   className?: string;
 }) => {
-  const TechCntClasses = twMerge(`p2 text-gray5 mr-[4rem] ${className}`);
+  const isMobile = useIsMobile();
+  const baseStyle = `w-full p2 text-gray5 `;
+
   return (
-    <p
-      className={TechCntClasses}
-      style={{
-        display: '-webkit-box',
-        wordWrap: 'break-word',
-        WebkitLineClamp: maxLines,
-        WebkitBoxOrient: 'vertical',
-        textOverflow: 'ellipsis',
-        overflow: 'hidden',
-      }}
+    <div
+      className={cn(`${isMobile ? 'max-w-[100vw]' : 'max-w-[80vw]'} ${className ? className : ''}`)}
```

Comment:
요런식으로 바꿀수 있을 것 같아용!!

```suggestion
    <div
      className={cn(isMobile ? 'max-w-[100vw]' : 'max-w-[80vw]'),className}
```

Reply:
className에 undefined 가 들어가는게 싫어서 저렇게 조건문으로 처리해놓긴 했어요 ㅜㅜ

Reply:
그럼 초기값을 `''` 로 지정해놓는 방법은 어떤가용??

Reply:
[ad22c08](https://github.com/dreamyPatisiel/devdevdev-client/pull/134/commits/ad22c080c100543762f7c54ae36303160f10459c) 😊👍